code/datums/spells/emplosion.dm

```diff
@@ -7,7 +7,11 @@
 
 	action_icon_state = "emp"
 
-/obj/effect/proc_holder/spell/targeted/emplosion/cast(list/targets, mob/user = usr)
+/obj/effect/proc_holder/spell/emplosion/create_new_targeting()
+	var/datum/spell_targeting/self/S = new()
+	return S
```

Comment:
I'll do it for the `/datum/spell_targeting/self` types. But I'll leave it for the others who use the default values so it can be tweaked more easily later. It does make sense for the self type since it has no parameters